README.md

```diff
@@ -54,14 +54,15 @@ See [remarkable.guide](https://remarkable.guide/tech/factory-reset.html) for mor
 ### Cloud API
 
 - (Unmaintained) [google-drive-remarkable-sync](https://github.com/bsdz/google-drive-remarkable-sync) - Apps Script API for reMarkable Cloud. Includes Synchronizer capability to automate mirroring of documents from Google Drive to reMarkable Cloud.
-- [jrmapi](https://github.com/jlarriba/jrmapi) - A Java API for the reMarkable Cloud.
+- (Likely Unmaintained) [jrmapi](https://github.com/jlarriba/jrmapi) - A Java API for the reMarkable Cloud.
```

Comment:
For unmaintained tags, we don't speculate. It's either unmaintained, or it's not marked.
```suggestion
- [jrmapi](https://github.com/jlarriba/jrmapi) - A Java API for the reMarkable Cloud.
```

Currently we have only added this to items where the repository has been archived, or the author has made an explicit statement that it is no longer maintained.

Reply:
Fair. But listing cloud clients that haven't been updated in the last 4 years also doesn't seem useful to me.

Reply:
It's still useful for those wanting a starting point.

Reply:
And how do we determine something is inactive? And who would be responsible for keeping the list maintained with that information?

Reply:
The thing is. I was looking for a working cloud client and this list hasn't been helpful at all, because none of the listed implementations seem to have any significant activity in the last years. In fact, the most recent client is actually rmapi, which is marked unmaintained. If I were looking for a starting point, this would be were you should look, but given the current state of this list it's the last thing I would look at, since the only project being marked as unmaintained.

Reply:
Trying to make this better here. If a project picks up speed, likely their maintainers or users will attempt to remove the tag via a PR, no?

Reply:
If you would like to add yet another thing that is maintained in this list, with clear criteria that can be discussed, please open an issue requesting it, where it can be properly discussed.

I don't believe your assumption that someone will open a PR to remove a tag if it picks up is actually realistic, as not everybody is looking at this list all the time, and often the maintainers aren't the ones adding things to this list, but others are. So they may not even be aware it's on here, let alone marked as inactive.